src/forms/dispense-form.component.tsx

```diff
@@ -18,24 +17,23 @@ import {
 } from "../medication-dispense/medication-dispense.resource";
 import { usePrescriptionDetails } from "../medication-request/medication-request.resource";
 import MedicationDispenseReview from "../components/medication-dispense-review.component";
-import { getPrescriptionDetailsEndpoint } from "../utils";
 
 interface DispenseFormProps {
   patientUuid: string;
   encounterUuid: string;
+  mutatePrescriptionTableRows: Function;
 }
 
 const DispenseForm: React.FC<DispenseFormProps> = ({
   patientUuid,
   encounterUuid,
+  mutatePrescriptionTableRows,
 }) => {
   const { t } = useTranslation();
   const isTablet = useLayoutType() === "tablet";
   const session = useSession();
-  const { requests, isError, isLoading } =
-    usePrescriptionDetails(encounterUuid);
+  const { requests, mutate, isLoading } = usePrescriptionDetails(encounterUuid);
```

Comment:
I realized instead of pulling the generic mutate function off the SWFConfig, I could just generic bound mutate, which seemed cleaner.